packages/syft/src/syft/service/request/request.py

```diff
@@ -807,16 +807,20 @@ def valid(self) -> Union[SyftSuccess, SyftError]:
         return SyftSuccess(message=f"{type(self)} valid")
 
     def mutate(self, obj: UserCode, context: ChangeContext, undo: bool) -> Any:
+        if "reason" in context.extra_kwargs.keys():
+            reason = context.extra_kwargs["reason"]
```

Comment:
Same here:
we can replace this with
reason = context.extra_kwargs.get("reason", ""), where default can empty string.